import_export/management/commands/export.py

```diff
@@ -0,0 +1,59 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from import_export.command_utils import (
+    get_default_format_names,
+    get_format_class,
+    get_resource_class,
+)
+
+
+class Command(BaseCommand):
+    help = "Export data from a specified resource or model in a chosen format."
+
+    def add_arguments(self, parser):
+        default_format_names = get_default_format_names()
+        parser.add_argument(
+            "format",
+            help=f"""Specify the export format. Can be one of the default formats
+            ({default_format_names}), or a custom format class provided as a dotted path
+            (e.g., 'XLSX' or 'mymodule.CustomCSV').""",
+        )
+        parser.add_argument(
+            "resource",
+            help="""Specify the resource or model to export. Accepts a resource class or
+            a model class in dotted path format "(e.g., 'mymodule.resources.MyResource'
+            or 'auth.User').""",
+        )
+        parser.add_argument(
+            "--encoding",
+            help="Specify the encoding to use for the exported data (e.g., 'utf-8').",
```

Comment:
Do we need to clarify that this will only apply to encoded data (i.e. not binary)

Reply:
in 0989f83